gcloud/connection.py

```diff
@@ -15,6 +15,7 @@
 """Shared implementation of connections to API servers."""
 
 import json
+import threading
```

Comment:
Actually this was already in the right order according to pylint. Moving it made it upset, so I moved it back.